pkgs/by-name/el/eloquent/package.nix

```
@@ -0,0 +1,87 @@
+{
+  stdenv,
+  lib,
+  fetchgit,
+  meson,
+  glib,
+  gjs,
+  ninja,
+  gtk4,
+  gsettings-desktop-schemas,
+  wrapGAppsHook4,
+  desktop-file-utils,
+  gobject-introspection,
+  glib-networking,
+  pkg-config,
+  libadwaita,
+  appstream,
+  blueprint-compiler,
+  gettext,
+  libportal-gtk4,
+  languagetool,
+  fasttext,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "eloquent";
+  version = "1.2";
+
+  # fetchFromGitHub did not fetch submodules
+  src = fetchgit {
+    url = "https://github.com/sonnyp/Eloquent.git";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-I4AQZl1zoZPhOwDR1uYNJTMRq5vQHPvyimC8OUAe+vY=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    desktop-file-utils
+    gobject-introspection
+    meson
+    ninja
+    pkg-config
+    wrapGAppsHook4
+    appstream
+    blueprint-compiler
+    gjs
+    libportal-gtk4
+    languagetool
+  ];
+
+  buildInputs = [
+    gettext
+    gjs
+    glib
+    glib-networking
+    gsettings-desktop-schemas
+    gtk4
+    libadwaita
+    libportal-gtk4
+  ];
+
+  postPatch = ''
+    substituteInPlace troll/gjspack/bin/gjspack --replace-fail "/usr/bin/env -S gjs" "${gjs}/bin/gjs"
```

```suggestion
    substituteInPlace troll/gjspack/bin/gjspack \
      --replace-fail "/usr/bin/env -S gjs" "${gjs}/bin/gjs"
```
Nit (same for the others).